index.d.ts

```diff
@@ -60,13 +60,25 @@ export type paypalLocalCodes =
 export type PayPalSupportedCurrency = paypalSupportedCurrencies
 export type PayPalLocaleCode = paypalLocalCodes
 
+export interface PostalAddress {
+    recipientName: string,
+    streetAddress: string,
+    extendedAddress: string,
+    locality: string,
+    countryCodeAlpha2: string,
+    postalCode: string,
+    region: string
+}
+
 export interface PaypalResponse {
     nonce: string,
     payerId: string,
     email: string,
     firstName: string,
     lastName: string,
-    phone: string
+    phone: string,
+    billingAddress: PostalAddress,
```

Comment:
Are these guaranteed to come back in the response?  What happens if a user doesn't have them set up?

I'm guessing it may need to be something like
`billingAddress?: PostalAddress`

same with the PostalAddress definition. I would assume `extendedAddress` would often come back as undefined

Reply:
actually looks like default is `null` (which really should be changed in the code base side but I can look at that later)

may end up having to be something like `extendedAddress: string | null,`

Reply:
I added string | null for all PostalAddress fields

In my iOS and Android tests, I never encountered a null reference exception on the native side, both billingAddress and shippingAddress objects seem to be available but filled with nulls 
(for example, Android `payPalAccountNonce.getShippingAddress()` never returned null, even if `shippingAddressRequired = false`)

Reply:
perfect 👍